denizen_scripts/relay/repo-link/Structure Data/RemoteDependencies/Text Events.dsc

```diff
@@ -9,14 +9,14 @@ MsgHover:
     type: procedure
     debug: false
     definitions: Hover|Text
-    description: Creates text with a hover message.
+    #description: Creates text with a hover message.
```

Comment:
These lines can be removed entirely; They're deprecated from an older script that referenced these keys specifically.